README.md

```diff
@@ -17,6 +17,10 @@ DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
 }
 ```
 
+## OUTPUT
```

Comment:
Thanks for the contribution! Minor nit pick but could you change the text `OUTPUT` to `Demo`?

Reply:
Sure thing :)